src/dlgProfilePreferences.cpp

```diff
@@ -157,12 +157,25 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
                                         .arg(tr("<p>Select the only or the primary font used (depending on <i>Only use symbols "
                                                 "(glyphs) from chosen font</i> setting) to produce the 2D mapper room symbols.</p>")));
     checkBox_isOnlyMapSymbolFontToBeUsed->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
-                                                             .arg(tr("<p>Using a single font is likely to produce a more consistent style but may "
-                                                                     "cause the <i>font replacement character</i> '<b>�</b>' to show if the font "
-                                                                     "does not have a needed glyph (a font's individual character/symbol) to represent "
-                                                                     "the grapheme (what is to be represented).  Clearing this checkbox will allow "
-                                                                     "the best alternative glyph from another font to be used to draw that grapheme.</p>")));
-
+                                        .arg("<p>Using a single font is likely to produce a more consistent style but may "
+                                             "cause the <i>font replacement character</i> '<b>�</b>' to show if the font "
+                                             "does not have a needed glyph (a font's individual character/symbol) to represent "
+                                             "the grapheme (what is to be represented).  Clearing this checkbox will allow "
+                                             "the best alternative glyph from another font to be used to draw that grapheme.</p>"));
+    checkBox_useWideAmbiguousEastAsianGlyphs->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                                                         .arg("<p>Some, generally East Asian, MUDs may use glyphs (characters) that are classified in Unicode "
```

Comment:
I think this needs to be simpler if we want non-native English speakers to be able to understand this.

Provide an example and say "should this character be drawn as 1 or 2 spaces wide"